### DISCUSSION TITLE
Why do some edit pages use AJAX and some don't?

### DISCUSSION THREAD

**Discussion by HenrijsS** [filamentphp/filament#6952] [Help] [answered]
I have to pretty simple forms.
Both of them have relationships, but one of them has a specific select fields that queries and formats the data returned.

But the weird part is that one of them, when saving, goes back to the table, but the other one stays there and saves the data using AJAX. The problem also arises that the updated_at doesn't change, even though the data is added.

**Comment by danharrin** [comment, marked as answer]
Both save via AJAX, but one responds with a redirect

**Reply by HenrijsS** [reply]
Missed the getRedirectUrl() function. Thanks!

**Reply by HenrijsS** [reply]
I was editing the children of the parent model and the childrens' timestamps are updating (Because they were changed), but the parent's model's timestamps only update when it's direct fields are changed.

Thought this was a thing with the [Table Repeater plugin](https://github.com/icetalker/filament-table-repeater), but the default repeater field also doesn't update parent timestamps when changed. Is this intended?

**Reply by danharrin** [reply]
You can use Laravel's `$touches` array on the model to touch parent timestamps

**Reply by HenrijsS** [reply]
Just what I needed. Thank you.
